### ISSUE TITLE
Possible Broker API implementations

### ISSUE THREAD

**Issue by OlegRa** [alpacahq/alpaca-trade-api-csharp#371]
@tstraus13 The [YAML file](https://github.com/alpacahq/bkdocs/blob/master/assets/openapi.yaml) for the Alpaca Broker API. I'm still thinking about possible relationships between this SDK and Alpaca Broker API. Each solution (same package, separate independent package, extracting some interfaces into the shared package) has its own pros and cons.

I'll add a separate issue for discussing this problem with the community. But in any case, these changes will be part of version 6.0 and I will do it only after implementing most of the other tasks planned for this major release.

_Originally posted by @OlegRa in https://github.com/alpacahq/alpaca-trade-api-csharp/issues/369#issuecomment-907669333_

**Comment by tstraus13** [comment]
Hello @OlegRa. I have just done a basic generation of the API using the YAML file with NSwag. I have attached the file to this post. Hopefully it is helpful as a starting point, if not feel free to not use it.
[openapiClient.cs.txt](https://github.com/alpacahq/alpaca-trade-api-csharp/files/7155722/openapiClient.cs.txt)

**Comment by OlegRa** [comment]
Extending the client SDK with the Broker API has no real value. Nobody has asked about it for years, so I think we can close this issue as outdated.
